dataland-frontend/src/assets/scss/components/form.scss

```diff
@@ -350,18 +350,17 @@ input[type="checkbox"], input[type="radio"]{
 }
 
 .p-inputswitch {
```

Comment:
Just out of curioustiy:  These changes make the input switch slider a little smaller relative to the rest of the elements on the page.

Have you done this just to make it visually more appealing?

Reply:
@MarcinStanczyk7  git blame says you have made this change.

Could you pls answer the question to me if possible?

Reply:
Yes I did it according to figma where it is smaller:

![image](https://github.com/d-fine/Dataland/assets/96523011/c84f99df-9b53-49eb-8229-c8c4dea35557)

Reply:
Ok cool thx 👍